### PR TITLE
Added demo script for testing CFEngine on a group of hosts

### PR DESCRIPTION
**Pull request by olehermanse**
I use this quite often when I want to test or demo something on a real setup.

### CODE REVIEW

File: docs/cf_remote_demo.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+cf-remote spawn --platform debian-7-x64 --count 1 --name debian7 --role client
+cf-remote spawn --platform debian-8-x64 --count 1 --name debian8 --role client
+cf-remote spawn --platform debian-9-x64 --count 1 --name debian9 --role client
+cf-remote spawn --platform rhel-5-x64 --count 1 --name rhel5 --role client
+cf-remote spawn --platform rhel-6-x64 --count 1 --name rhel6 --role client
+cf-remote spawn --platform rhel-7-x64 --count 1 --name rhel7 --role client
+
+cf-remote --version master install --demo --bootstrap hub --hub hub
+
+cf-remote deploy --hub hub /northern.tech/cfengine/masterfiles
```
Comment (vpodzime) [inline docs/cf_remote_demo.sh:20]: Would be better to use a relative path here.

File: docs/cf_remote_demo.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+cf-remote --version master install --demo --bootstrap hub --clients ubuntu16
+cf-remote --version master install --demo --bootstrap hub --clients ubuntu14
+cf-remote --version master install --demo --bootstrap hub --clients centos6
+cf-remote --version master install --demo --bootstrap hub --clients centos7
+cf-remote --version master install --demo --bootstrap hub --clients debian7
+cf-remote --version master install --demo --bootstrap hub --clients debian8
+cf-remote --version master install --demo --bootstrap hub --clients debian9
+cf-remote --version master install --demo --bootstrap hub --clients rhel5
+cf-remote --version master install --demo --bootstrap hub --clients rhel6
+cf-remote --version master install --demo --bootstrap hub --clients rhel7
```
Comment (vpodzime) [inline docs/cf_remote_demo.sh:32]: Hmm, would `ubuntu18,ubuntu16,ubuntu14,...` work?

File: docs/cf_remote_demo.sh [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+cf-remote sudo -H centos6 "service iptables stop"
```
Comment (vpodzime) [inline docs/cf_remote_demo.sh:34]: I think *centos7* needs `systemctl stop firewalld`.

Comment (vpodzime) [reply docs/cf_remote_demo.sh:34]: And so does *rhel7* and *rhel6* (iptables).

**Review by vpodzime — COMMENTED** [review]
Looks good to me otherwise.

File: docs/cf_remote_demo.sh (hunk 1 above)
Comment (olehermanse) [reply docs/cf_remote_demo.sh]: Done.

File: docs/cf_remote_demo.sh (hunk 2 above)
Comment (olehermanse) [reply docs/cf_remote_demo.sh:35]: Yes, but I find it handy that it's easy to comment out lines.

**Comment by olehermanse** [comment]
@vpodzime I created this bug, because of the new suse rpms, this setup no longer works, cf-remote needs some adjustments before we merge this:

https://tracker.mender.io/browse/CFE-3445

**Comment by olehermanse** [comment]
Fix for above mentioned bug: https://github.com/cfengine/core/pull/4335

File: docs/cf_remote_demo.sh (hunk 3 above)
Comment (olehermanse) [reply docs/cf_remote_demo.sh:37]: @vpodzime With the AMIs we use, those are not needed. This works (just tested it, all hosts reporting). So I don't think I need to add more commands.

**Review by vpodzime — APPROVED** [review]
ACK

**Pull request merged**